### ISSUE TITLE
[A11y Library Fix] - `No data` text doesn't meet required minimum contrast ratio

### ISSUE THREAD

**Issue by jakewheeler** [CDCgov/dibbs-ecr-viewer#401]
## What needs to be done

When the `No data` text appears within the eCR Library table due to a lack of `Encounter Date` information, it will not meet the required minimum contrast ratio when the row has a gray background. We should consider making this text a darker color so that the minimum contrast ratio requirement is always met regardless of whether a table row has a white or gray background.

Please see the Axe [issue report ](https://cdc-devtools.dequecloud.com/issues/80c530bc-a498-4325-84f3-0e84851bb08e) for more details.

## Why it needs to be done

Low color contrast can make it harder for users to read this text within the eCR Library.

## Acceptance Criteria

- [ ] `No data` text color has been made darker, issue is no longer reported by the Axe extension
- [ ] Designer has reviewed and approved the updated text color

## Technical Notes

The [noData](https://github.com/CDCgov/dibbs-ecr-viewer/blob/main/containers/ecr-viewer/src/app/utils/data-utils.tsx#L17) element currently makes use of `text-base` for its color. We may want to consider using `text-base-darker` here instead.
